x-pack/plugins/canvas/public/components/es_data_view_select/es_data_view_select.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DataView } from '@kbn/data-views-plugin/common';
+import { sortBy } from 'lodash';
+import React, { useRef, useState } from 'react';
+import useEffectOnce from 'react-use/lib/useEffectOnce';
+import { useDataViewsService } from '../../services';
+import {
+  ESDataViewSelect as Component,
+  ESDataViewSelectProps as Props,
+} from './es_data_view_select.component';
+
+type ESDataViewSelectProps = Omit<Props, 'indices' | 'loading'>;
+
+export const ESDataViewSelect: React.FunctionComponent<ESDataViewSelectProps> = (props) => {
```

Comment:
Nit: Not necessary for this PR, but I'm wondering if we should standardize the way we define functional components as part of that style guide we wrote. I usually do:

``` typescript
export const ESDataViewSelect = ({ value, onChange }: ESDataViewSelectProps) => {
  ...
}
```

Instead of how it's done here. I find that that is more concise and easier to understand, but it isn't explicity labeled as a `function component`, the dev has to infer that based on the fact that it returns JSX. Wondering if we should come together to use a standard definition, for all of our code, or if we should just define it in the same way it's defined elsewhere in the plugin.

Reply:
I actually prefer using the FC type like this:

```javascript
import { FC } from 'react';

export const ESDataViewSelect: FC<ESDataViewSelectProps> = ({ value, onChange }) => {
  ...
}
```

which only adds two characters while also being easy to read/concise.

Reply:
Sounds like a good pattern!